src/toxcore/dht/server/mod.rs

```diff
@@ -432,7 +432,7 @@ impl Server {
             payload
         ));
 
-        self.send_to(target_peer.saddr, nodes_req)
+        self.send_to_node(target_peer, nodes_req)
```

Comment:
c-toxcore sends two packets with different ping ids so it will process both responses. If we send same packet we will be able to process only one response.

Reply:
NodesRequest sends packets with different ping_id to both IPv4 and IPv6.
PingRequest is not sending packet to both addresses.
NatPingRequest uses same ping_id to both addresses.